WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -0,0 +1,78 @@
+  <table class="table registrations-table selectable-rows">
+    <thead>
+      <tr>
+        <th>WCA id</th>
+        <th>Name</th>
+        <th>Country</th>
+        <th>Birthday</th>
+        <th>Guests</th>
+        <th>Comments</th>
+        <% @competition.events.each do |event| %>
+          <th>
+            <span class="cubing-icon icon-<%= event.id %>" data-toggle="tooltip" data-placement="top" title="<%= event.name %>"></span>
+          </th>
+        <% end %>
+      </tr>
+    </thead>
+    <tbody>
+      <% @competition.registrations.order(status: :desc).each do |registration| %>
+        <tr class="registration-<%= registration.status %>" id="registration-<%= registration.id %>">
+          <td>
+            <%= registration.personId %>
+          </td>
+          <td>
+            <a href="mailto:<%= registration.email %>" data-toggle="tooltip" data-placement="left" data-container="body" title="<%= registration.email %>"><%= registration.name %></a>
+          </td>
+          <td>
+            <%= registration.countryId %>
+          </td>
+          <td>
+            <%= registration.birthday %>
+          </td>
+          <td class="guests" data-toggle="tooltip" data-placement="left" data-container="body" title="<%= registration.guests %>">
+            <%= registration.guests %>
+          </td>
+          <td class="comments" data-toggle="tooltip" data-placement="left" data-container="body" title="<%= registration.comments %>">
+            <%= registration.comments %>
+          </td>
+          <% @competition.events.each do |event| %>
+            <td>
+              <span class="event-checkbox">
+                <label>
+                  <%= check_box "competition[event_ids]", event.id, { checked: registration.events.include?(event), disabled: true } %>
+                  <span class="cubing-icon icon-<%= event.id %>" data-toggle="tooltip" data-placement="top" title="<%= event.name %>"></span>
+                </label>
+              </span>
+            </td>
+          <% end %>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+  <div class="btn-group" role="group">
+    <a href="<%= registrations_path(format: :csv) %>" class="btn btn-info">
+      <span class="glyphicon glyphicon-floppy-save"></span>
+      Export to CSV
+    </a>
+    <a href="#" id="email-selected" target="_blank" class="btn btn-info selected-registrations-actions">
+      <span class="glyphicon glyphicon-envelope"></span>
+      Email
+    </a>
+    <button type="submit" class="btn btn-success selected-pending-registrations-actions" name="registrations_action" value="accept-selected">
+      <span class="glyphicon glyphicon-ok"></span>
+      Approve
+    </button>
+    <button type="submit" class="btn btn-warning selected-approved-registrations-actions" name="registrations_action" value="reject-selected">
+      <span class="glyphicon glyphicon-remove"></span>
+      Reject
```

Comment:
So I'm wondering what the point of "rejecting" registrations is, and/or if we should change vocabulary here. Does "Un-accept" or "Revert to pending" make more sense?